test/pc_subjective_model_test.py

```diff
@@ -22,26 +22,26 @@ def setUp(self):
     def test_btnr_subjective_model(self):
         subjective_model = BradleyTerryNewtonRaphsonPairedCompSubjectiveModel(self.pc_dataset_reader)
         result = subjective_model.run_modeling(zscore_output=True)
-        self.assertAlmostEquals(float(np.sum(result['quality_scores'])), 0, places=4)
-        self.assertAlmostEquals(float(np.var(result['quality_scores'])), 1, places=4)
+        self.assertAlmostEqual(float(np.sum(result['quality_scores'])), 0, places=4)
```

Comment:
There was a warning to use `assertAlmostEqual` (singular) instead of `assertAlmostEquals` (plural)